### ISSUE TITLE
Replacing the value inside an element

### ISSUE THREAD

**Issue by computersrmyfriends** [No9/harmon#48]
Hi,
Thanks for your wonderful lib.
I'm struggling with replacing the value inside a div.

Here's an example of what I'm facing:

```
<div>hello</div>
<div>world</div>
```

The div's don't have id or class. I just need to replace the hello with hi after selecting the div based on the content inside the element. I need to achieve this somehow. I would appreciate any help.

Thanks

**Comment by computersrmyfriends** [comment]
Yes but div would select both div's. I wanted to select omly the div having the value hello in it.

Thanks

**Comment by No9** [comment]
Hi @computersrmyfriends 
You would have to test for that in the `func` based on the properties of the `node` parameter
